nerfstudio/data/datamanagers/full_images_datamanager.py

```diff
@@ -254,8 +251,7 @@ def cache_images(self, cache_images_option):
                     mask = cv2.fisheye.undistortImage(mask, K, distortion_params, None, newK)
                     data["mask"] = torch.from_numpy(mask).bool()
                 K = newK
-            else:
-                raise NotImplementedError("Only perspective and fisheye cameras are supported")
```

Comment:
We may still want to keep this behavior per earlier comment.

Reply:
Thank you! Indeed, I removed it for some testing and haven't noticed. Put it back.